homeassistant/components/light/mqtt_json.py

```diff
@@ -50,6 +61,10 @@
     vol.Optional(CONF_OPTIMISTIC, default=DEFAULT_OPTIMISTIC): cv.boolean,
     vol.Optional(CONF_BRIGHTNESS, default=DEFAULT_BRIGHTNESS): cv.boolean,
     vol.Optional(CONF_RGB, default=DEFAULT_RGB): cv.boolean,
+    vol.Optional(CONF_RGB_VALUE_TEMPLATE, default=DEFAULT_RGB_VALUE_TEMPLATE):
```

Comment:
The None check as it was wouldn't work here. The default value was the identity function which would in this context just return the JSON as is and try to convert it as int later which obviously won't work.

Or are you suggesting that I do the defaulting to the template value inside the function instead of in the config values? What is the difference?